builtin/mainmenu/modmgr.lua

```diff
@@ -395,6 +395,11 @@ function modmgr.installmod(modfilename,basename)
 
 	modmgr.refresh_globals()
 
+	if basefolder.type == "invalid" then
+		return
```

Comment:
if this can return false rather than nil, then this if-else can be replaced with:

```lua
return basefolder.type ~== "invalid"
```